examples/website/src/shared/route/blockAdapters.ts

```diff
@@ -111,7 +117,7 @@ function getBlocks(blocks: any, articleMeta?: ArticleMeta): Block[] {
           title: block.title,
           lead: block.lead
         }
-        if (articleMeta && (index == 0 || (hasTitleImage && index == 1))) {
+        if (articleMeta && (index === 0 || (hasTitleImage && index === 1))) {
```

Comment:
This change doesn't really belong to this PR or why did you change this?

Reply:
its basically a mistake, but will revert for now